src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,90 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import { FC } from 'react';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const countOfPage = Math.ceil(total / perPage);
+  const isFirstPage = currentPage === 1;
+  const isLastPage = currentPage === countOfPage;
+
+  const pages: number[] = [];
+
+  for (let i = 1; i <= countOfPage; i += 1) {
+    pages[i - 1] = i;
+  }
+
+  const handleClickOnRightButton = () => {
+    if (currentPage < countOfPage) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  const handleClickOnLeftButton = () => {
+    if (!isFirstPage) {
+      onPageChange(currentPage - 1);
+    }
```

Comment:
it works just fine, but if you add more logic inside 'if' block, it would be messy, instead, if current page is first, you can return from function, otherwise execute code below
```suggestion
    if (isFirstPage) {
      return;
    }
    
    onPageChange(currentPage - 1);
```